### PR TITLE
[TW-305] Secret vars update

### PR DESCRIPTION
**Pull request by claireriley**
**Managing environments**
* fix screenshots
* add more info about secret variables
* minimum necessary edits in spots I was editing for secret vars, full rewrite needed (created TW-335)

**Using variables**
* fix screenshots

**Defining roles**
* add environment section (this isn't in the webdash so kept extremely simple, reaching out to see if it can be added)

### CODE REVIEW

File: src/pages/docs/sending-requests/managing-environments.md [hunk 1]
```diff
@@ -76,64 +63,64 @@ You can use the current value of your environment variables to ensure that sensi
 
 You will see the selected environment status at the top-right of Postman, with any _active_ environment selected in the drop-down. In __Environments__ on the left you can access all environments, and click the check-mark button to make an environment active. Select an environment to open it for editing in a tab.
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/managing-environments.md:64]: There are some "clicks" on this page you can change to "select" if you want 🙂

File: src/pages/docs/sending-requests/managing-environments.md [hunk 2]
```diff
@@ -152,7 +139,7 @@ You can use the same variable notation in request URLs, parameters, headers, and
 Hover over a variable reference to see its current value.
 
 > If more than one variable with the same name is available to a request, Postman will use the value from the variable with narrowest [scope](/docs/sending-requests/variables/#variable-scopes). This means that if you have an environment variable with the same name as a collection or global variable, Postman will use the environment variable, but local and data variable values will supersede environment values. The value of any overridden variables will display with a strikethrough.
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/managing-environments.md:141]: Add a </br> after this paragraph to create some white space before image?

Also, I didn't know you could put an image in a note--cool!!

File: src/pages/docs/sending-requests/managing-environments.md [hunk 3]
```diff
@@ -168,17 +155,17 @@ You can edit variables either by opening an environment from __Environments__ on
 
 > You will only be able to edit environments where you have [editor access](#requesting-environment-access).
 
-[![Edit environment](https://assets.postman.com/postman-docs/edit-env-vars-v8.jpg)](https://assets.postman.com/postman-docs/edit-env-vars-v8.jpg)
+[![Edit environment](https://assets.postman.com/postman-docs/environment-editor-new-v9.13.jpg)](https://assets.postman.com/environment-editor-new-v9.13.jpg)
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/managing-environments.md:158]:
```suggestion
[![Edit environment](https://assets.postman.com/postman-docs/environment-editor-new-v9.13.jpg)](https://assets.postman.com/postman-docs/environment-editor-new-v9.13.jpg)
```

**Review by mark-anthony-postman — APPROVED** [review]
Made a few small comments, but otherwise LGTM. Great updates throughout! 🌟

**Review by HeatherStenson — APPROVED** [review]
Looks great! 🍊

File: src/pages/docs/sending-requests/managing-environments.md (hunk 2 above)
Comment (HeatherStenson) [reply src/pages/docs/sending-requests/managing-environments.md:141]: I didn't know this either! Tucking this one away for the future 🎉

File: src/pages/docs/sending-requests/managing-environments.md (hunk 1 above)
Comment (claireriley) [reply src/pages/docs/sending-requests/managing-environments.md]: Will do, only hesitated because I was worried it'd open a can of worms 😆

**Pull request merged**